webapp/utils/virtual_terminal.py

```diff
@@ -17,7 +17,7 @@
     import termios      # used to set the window size (look up "TIOCSWINSZ" in https://linux.die.net/man/4/tty_ioctl)
     import fcntl        # I/O for file descriptors; used for setting terminal window size
 
-# pylint: disable=invalid-name
+# pylint: disable=dangerous-default-value
```

Comment:
we wouldn't need this if you treat `term_cmd` parameter to `init_connect()` as a positional argument. Looks like [we're already specifying it as a positional argument](https://github.com/DVC-Viking-Robotics/webapp/blob/73b9aaf7bf350af50894e2eeab774d1663facffb/webapp/sockets.py#L193) during the `init_connect()` call in `sockets.py` anyway